### ISSUE TITLE
Reproduced Results

### ISSUE THREAD

**Issue by liguopeng0923** [bwconrad/flexivit#4]
Hi, @bwconrad ,

Great work.
I want to know how to reproduce the results of your teaser.
<img width="494" alt="image" src="https://github.com/bwconrad/flexivit/assets/61257834/fd4336e2-f25b-4104-bb87-9367066283f7">

An image is splited to 2*2, and the accuracy is 84.4%.

Best,
Guopeng.

**Comment by bwconrad** [comment]
What this figure is visualizing is the process of patchifying an image with different patch sizes using pi-resizing and having a much less significant performance drop compared to naive approaches. 

The image being split 2x2 and 3x3 is just for visualization purposes. It represents the idea of, for example, using a patch size of 16 and 32 on the same image to adjust the number of tokens being processed.

I am not an author of this work, this is simply a reimplementation of the architecture. If you want to try and reproduce the results just follow the hyperparameters listed in the paper and use your own classification training script. The accuracy numbers from the figure look to be for a ViT-L at patch sizes of 24 and 10 according to Table 2 in the appendix.
